### DISCUSSION TITLE
Which service to use for our forum?

### DISCUSSION THREAD

**Discussion by martinjagodic** [decaporg/decap-cms#6720] [Ideas]
Netlify CMS used the Discourse subforum on https://answers.netlify.com/, which is not an option for Decap as we are not affiliated with Netlify anymore.

We cannot move content from the old forum since users never agreed to have their posts sent into a new system. That means we don't have to worry about migration, but we lose some valuable resources.

### Requirements
- service for long-term/forum communications (we have Discord for short-term/chat)
- easy to configure & manage
- easy to use
- SEO: ranks high in search results when users seek answers to problems
- code-friendly: supports markdown and code snippets

### Github Discussions
We propose to use this service, as it addresses majority of our needs. There are services with more features (like Discourse), but do we really need all that?

#### Pros
- easy & free to configure & manage
- we already use them to some extent
- easy to convert from/to issues
- no need for another login

#### Cons
- potentially confusing when to use Issues vs Discussions
- more vendor lock-in

### What do you think? 

Do you see any pitfalls with this service, is there a better one? Would you add anything else to requirements?

**Comment by mortendk** [comment]
Just go with GH - it works for so many other projects, and even better it gives me only one place to go to figure out stuff :) 
If it works for you and the rest of the team that takes the heavy lifting then go with it

**Comment by pktiuk** [comment]
Using just Github Discussions would save you a lot of time. I think It is a good option.    

Another potential problem with usage of GH Discussions may appear, when you will have multiple repositories and each of them will have separate discussions. This may confuse users.

**Comment by VitroidFPV** [comment]
I find navigating most forums clunky at times, GH Discussions are pretty good and it would consolidate it nicely into one place. That would be a double-edged sword though, (as @pktiuk mentioned) that when/if there are new repos added, it may get a bit messy

**Comment by martinjagodic** [comment]
Github Discussions it is. The concern about confusion across repositories is valid. For now, we will avoid that by disabling them on all repositories except this one.

**Reply by mmkal** [reply]
Will the discord continue to exist?

**Reply by samo4** [reply]
absolutely

**Reply by pktiuk** [reply]
Github Discussions are a bit more like stack overflow for this app.    

In general, I think some kinds of discussions should be limited on discord and redirected here (like discussions about usage problems/how to do something etc.), because discord is not accessible from wider internet and is not very searchable.
